VideoWeb/VideoWeb/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -4356,6 +4357,7 @@ export class ParticipantForUserResponse implements IParticipantForUserResponse {
             if (Array.isArray(_data['linked_participants'])) {
                 this.linked_participants = [] as any;
                 for (let item of _data['linked_participants']) this.linked_participants!.push(LinkedParticipantResponse.fromJS(item));
+                this.hasInterpreterLink = this.linked_participants.some(x => x.link_type === LinkType.Interpreter);
```

Comment:
was this added automatically?

Reply:
nope added it so you can see if the LinkedParticipants have an INterpeter Link

Reply:
we cant add to here manually because when we call the update client command it will be overwritten :(

Reply:
file is autogenerated

Reply:
removed due to the apiclient being auto generated, sorry didn't realise as was only looking at the bit of code I changed.